### ISSUE TITLE
FileNotFoundError: Installing version 0.10.2 with pip fails

### ISSUE THREAD

**Issue by nsandman09** [jookies/jasmin#905]
Here is the output of running `pip3.8 install jasmin` on Oct 18, 2020, where the latest release [here](https://pypi.org/project/jasmin/#history) is 0.10.2:

```
ERROR: Command errored out with exit status 1:
     command: /usr/local/opt/python@3.8/bin/python3.8 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/h2/615063vn3gx1pxs1y6qz_dzm0000gp/T/pip-install-vri20czq/jasmin/setup.py'"'"'; __file__='"'"'/private/var/folders/h2/615063vn3gx1pxs1y6qz_dzm0000gp/T/pip-install-vri20czq/jasmin/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /private/var/folders/h2/615063vn3gx1pxs1y6qz_dzm0000gp/T/pip-pip-egg-info-vwr7wvxz
         cwd: /private/var/folders/h2/615063vn3gx1pxs1y6qz_dzm0000gp/T/pip-install-vri20czq/jasmin/
    Complete output (7 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/h2/615063vn3gx1pxs1y6qz_dzm0000gp/T/pip-install-vri20czq/jasmin/setup.py", line 22, in <module>
        install_requires=parse_requirements('requirements.txt'),
      File "/private/var/folders/h2/615063vn3gx1pxs1y6qz_dzm0000gp/T/pip-install-vri20czq/jasmin/setup.py", line 5, in parse_requirements
        lineiter = (line.strip() for line in open(filename))
    FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

However, if I run `pip2 install jasmin==0.9.31`, it works without errors.

The same FileNotFoundError occurs in different versions of Python, including 2.7. It also happens in both my Debian Docker image (buster) and on macOS Catalina (10.15.7). This makes me think it's not an issue with my setup, but if any more information is needed to rule that out, I'd be happy to provide it.

Also, I didn't save the results as I tried this a few days ago, but manually downloading the files and running setup.py yielded a different error. If that would be helpful to pinpoint the cause of this problem, I can provide that output to you too.

Thanks in advance!

**Comment by farirat** [comment]
I confirm the issue with the packaging, work in progress to fix it.

**Comment by farirat** [comment]
Issue fixed in 0.10.5
